### PR TITLE
Updated README.md file with latest technology stack

### PR DESCRIPTION
**Pull request by MSS-Shalavritha**

### CODE REVIEW

**Review by Gajjela-Koushik-Reddy — COMMENTED** [review]
Looks good to me too, we can add more as we move forward in the project.

**Review by arshiya19 — COMMENTED** [review]
Looks good to me too, we can add more as we move forward in the project.

File: README.md [hunk 1]
```diff
@@ -25,16 +25,18 @@ In this project, we have implemented food pantry management using a NodeJS-based
 
 > Back-End
 
-- MongoDB
 - Express JS
-- JavaScript
+- TypeScript
+- PostGres
```
Comment (Jooms) [inline README.md:30]: PostgreSQL

**Review by Jooms — DISMISSED** [review]

**Review by Jooms — APPROVED** [review]

**Review by TJSidhu1 — APPROVED** [review]
LGTM!

**Pull request merged**